adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -34,18 +34,31 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
+class WiFiConnType: # pylint: disable=too-few-public-methods
+    """An enum-like class representing the different types of WiFi connections
+    that can be made. The values can be referenced like ``WiFiConnType.normal``.
+    Possible values are
+    - ``ThermocoupleType.normal``
+    - ``ThermocoupleType.enterprise``
```

Comment:
Change to `WiFiConType.enterprise`

Reply:
This class is being removed in an upcoming commit and replaced with a couple of constants.